cloud-functions/upload_fhir_bundle/test_upload_fhir_bundle.py

```diff
@@ -54,15 +59,14 @@ def test_upload_fhir_bundle_bad_body():
         "bundle": {"resourceType": "Bundle"},
     }
 
-    assert upload_fhir_bundle(request) == {
-        "description": {
-            "loc": ["dataset_id"],
-            "msg": "str type expected",
-            "type": "type_error.str",
-        },
-        "status": 400,
-        "summary": "Invalid request body",
-    }
+    result = upload_fhir_bundle(request)
+    expected_result = fail(
+        "{'loc': ['dataset_id'], 'msg': 'str type expected', 'type': 'type_error.str'}",
+        "Bad Request",
+    )
+    expected_result.status_code = 400
```

Comment:
PR 37 currently replaces success with make_response, and `status` is replaced with `status_code.